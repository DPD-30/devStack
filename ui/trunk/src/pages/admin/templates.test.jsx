import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesAdminTemplates from './templates.jsx'

jest.unmock('./templates.jsx')


describe('', () => {
    it('PagesAdminTemplates should match snapshot', () => {
        const comp = renderToString(<PagesAdminTemplates />)

        expect(comp).toMatchSnapshot()
    })
})
