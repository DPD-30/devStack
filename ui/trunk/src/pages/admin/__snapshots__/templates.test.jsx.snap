// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesAdminTemplates should match snapshot 1`] = `
<div data-reactroot>
  Admin: Templates
</div>
`;
